gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/exclusion/ReduceExclusion.java

```diff
@@ -50,29 +50,30 @@ public ReduceExclusion(GradoopId startId) {
   }
 
   /**
-   * Creates a new logical graph that contains only vertices and edges that
-   * are contained in the starting graph but not in any other graph that is part
-   * of the given collection.
+   * Reduces the starting graph to contain only vertices and edges that are not contained in any
```

Comment:
Good description. Please move it from the execute method to the class javadoc. See #1240